datamade/how-to
# updated django translation on heroku, add wagtail instructions

## Overview

Updates translation documentation to include a new pattern and Wagtail setup instructions

Handles #225 

### Notes

Once we have a pattern for extending translations in Django to React code, we will open a new PR 

## Testing Instructions

* read the docs and confirm the approach makes sense
* confirm links and formatting work
* helpful plugin for previewing GitHub Markdown: https://marketplace.visualstudio.com/items?itemName=bierner.github-markdown-preview
